packages/components/src/tooltip/index.native.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+// for the native mobile, just shortcircuit the Tooltip to return its child
+export default ( props ) => React.Children.only( props.children );
```

Comment:
This doesn't look like a tooltip implementation at all :)

Reply:
@hypest did this changes I think it's just a stub implementation to avoid changes on the format-toolbar component that uses this. To be honest tooltips don't make a lot of sense on native apps. You are not hovering any elements on them...